content/blog/2016/2016-03-02-gsoc2016-announcement.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Jenkins participates in Google Summer of Code 2016"
+tags:
+- general
+- jenkins2
+- gsoc
+author: oleg_nenashev
+---
+
+We are happy to announce that Jenkins project application has been accepted to [Google Summer of Code 2016](https://developers.google.com/open-source/gsoc/timeline). 
+Thanks to everybody who helped us to prepare the application and project ideas!
+
+This year we invite students to join the Jenkins community and to work together on the ongoing Jenkins 2.0 activities and other medium-term projects.
+
+In GSoC student projects we want to address the user experience in wide terms. 
+It includes not only user interface changes, but also availability improvements and major new features like [Pipeline-as-code](https://wiki.jenkins-ci.org/display/JENKINS/2.0+Pipeline+as+Code). 
+The activity involves all components of Jenkins: core, plugins, website and internal automation infrastructure.
+
+Our [project ideas](https://wiki.jenkins-ci.org/display/JENKINS/Google+Summer+Of+Code+2016) include:
```

Comment:
I think this should be rephrased as "Some of our suggested project ideas"

IIRC with GSoC projects can suggest projects but students are also encouraged to present their own projects.